language-support/hs/bindings/test/DA/Ledger/Tests.hs

```diff
@@ -300,6 +301,15 @@ tGetActiveContracts withSandbox = testCase "tGetActiveContracts" $ run withSandb
         assertEqual "active" ev' active
         -- assertEqual "active" ev active -- TODO: enable if this should be true & we get a fix
 
+tGetLedgerConfiguration :: SandboxTest
+tGetLedgerConfiguration withSandbox = testCase "tGetLedgerConfiguration" $ run withSandbox $ \_pid -> do
+    lid <- getLedgerIdentity
+    xs <- getLedgerConfiguration lid
+    Just(Right x) <- liftIO $ timeout 1 (takeStream xs)
+    liftIO $ do
+        --print x
+        assertEqual "defined" x x -- check nothing is undefined
```

Comment:
Can we come up with a more useful thing to test here? If you really just want to test that no fields are bottom, I would go for `evaluate $ force x` (you will need to add a `NFData` instance). to at least make the intent clearer than comparing a value to itself.